jupyter-docker/spark-defaults-temp.conf

```diff
@@ -0,0 +1,28 @@
+#######################
+# required by DataProc
+#######################
+
+# Uses the YARN configuration in $HADOOP_CONF_DIR
+
+spark.master yarn
+spark.submit.deployMode client
+
+###################
+# required by Hail
+###################
+
+# Distributes from master node to the working directories of executors
+
+spark.jars /spark/spark-2.0.2/gcs-connector.jar,/hail/hail-hail-is-master-all-spark2.0.2-HAILHASH.jar
```

Comment:
Add a note here about how `HAILHASH` is substituted.  I would also prefer making it look more like something that gets substituted, maybe `[HAILHASH]`